tests/test_user.py

```diff
@@ -111,6 +112,190 @@ def test_change_middle_name(self, driver, profile_settings_page, fake):
+                email_body = EmailAccess.get_latest_email_body_by_imap(
+                    settings.IMAP_HOST,
+                    settings.IMAP_EMAIL,
+                    settings.IMAP_EMAIL_PASSWORD,
+                    'Inbox',
+                    'SUBJECT',
+                    'Confirm account merge',
+                )
+                # Search through the email body and verify that the OSF account owner's
+                # email address is in the body of the email
+                assert str(email_body).find(settings.USER_ONE) > 0
+        finally:
+            # Lastly delete the unconfirmed email from the account
+            unconfirmed_email = settings_page.get_unconfimred_email_item(
+                settings.IMAP_EMAIL
+            )
+            if unconfirmed_email is not None:
+                delete_button = unconfirmed_email.find_element_by_css_selector(
+                    'button[data-test-delete-button]'
+                )
+                delete_button.click()
+                # Verify email address in text of confirmation modal
+                assert (
+                    settings_page.confrim_remove_email_modal.deleted_email.text
+                    == settings.IMAP_EMAIL
+                )
+                settings_page.confrim_remove_email_modal.delete_button.click()
+                settings_page.reload()
+                unconfirmed_email = settings_page.get_unconfimred_email_item(
+                    settings.IMAP_EMAIL
+                )
+                assert unconfirmed_email is None
+
+    def test_user_account_settings_storage_locations(self, driver, session):
+        """Check the list of storage locations in the dropdown listbox on the User
+        Account Settings page and verify that the correct default storage location
+        is selected.
+        """
+        settings_page = user.AccountSettingsPage(driver)
+        settings_page.goto()
+        assert user.AccountSettingsPage(driver, verify=True)
+        settings_page.scroll_into_view(settings_page.storage_location_listbox.element)
+        # Use the api to get the logged in user's default region and verify that it
+        # matches the storage location region that is displayed as selected in the
+        # listbox
+        user_region = osf_api.get_user_region_name(session)
+        assert settings_page.storage_location_listbox.text == user_region
+        # Click the listbox to display all of the storage locations and get a list
+        # of the displayed regions
+        settings_page.storage_location_listbox.click()
+        listbox_items = settings_page.storage_location_listbox.find_elements(
+            By.CSS_SELECTOR,
+            'div.ember-basic-dropdown-content-wormhole-origin > div > ul > li',
+        )
+        listbox_regions = []
+        for region in listbox_items:
+            listbox_regions.append(region.text)
+        listbox_regions.sort()
```

Comment:
golfier: `listbox_regions = sorted([region.text for region in listbox_items])`

Reply:
Fixed